configs/woocommerce-github-sync-labels/README.md

```diff
@@ -0,0 +1,18 @@
+# woocommerce-github-sync-labels config
+
+In Grow we adhere to Woo convention of GitHub labels (PbIy4N-2wg-p2). However, we add a few more.
+
+This folder contains configs to be used with [`woocommerce-github-sync-labels`](https://github.com/woocommerce/woocommerce-github-sync-labels) to automatically, sync those labels with all our repos.
+
+## How to use
+
+1. [Install and setup `woocommerce-github-sync-labels`](https://github.com/woocommerce/woocommerce-github-sync-labels#install)
+2. Copy configs from this folder into the cloned tool folder 
+3. Run dry run
+	```
+	npm start -- --dry-run --preserve-labels
+	```
```

Comment:
❓ Just curious, why we need to add `--dry-run` with `npm start`  ??

Reply:
Ok, I can see in https://github.com/woocommerce/woocommerce-github-sync-labels#install.

This is for testing? I think `npm test` also works?

Reply:
Yes, this is for testing. Given the operation is not easily reversible, I think the dry run/simulation is valuable.